interface-definitions/include/constraint/interface-name.xml.in

```diff
@@ -0,0 +1,4 @@
+<!-- include start from constraint/interface-name.xml.in -->
+<regex>(bond|br|dum|en|ersp|eth|gnv|lan|l2tp|l2tpeth|macsec|peth|ppp|pppoe|pptp|sstp|tun|vti|vtun|vxlan|wg|wlan|wwan)[0-9]+(.\d+)?|lo</regex>
```

Comment:
We also have virtual-ethernet `veth` interfaces.
```
set interfaces virtual-ethernet veth0 peer-name veth1
```

Reply:
Good catch, @sever-sever ; I'll update.